### DISCUSSION TITLE
Getting Client Cvar

### DISCUSSION THREAD

**Discussion by BlackYuzia** [saul/demofile#307] [Questions] [answered]
Hi @saul 
Did you know, can I get, for example, get value of **sensivity** cvar of client (or any else client cvar)?
That is **client cvar** (not a server convar), so I don't sure what server ~~путин~~ put in demo client convars.

**Comment by saul** [comment, marked as answer]
Unfortunately client cvar values aren't recorded in demos - including in-eye demos.
